src/UIContainer.ts

```diff
@@ -910,9 +918,34 @@ export class UIContainer extends HTMLElement {
         return node === this || this._playerEl.contains(node);
     }
 
-    private readonly _onKeyUp = (): void => {
+    private readonly _onKeyDown = (event: KeyboardEvent): void => {
+        if (this.deviceType === 'tv') {
+            if (isBackKey(event.keyCode)) {
+                this.setUserIdle_();
+                return;
+            }
+
+            const focusedChild = getFocusedChild(getFocusableChildren(this));
```

Comment:
Nit-pick: store `getFocusableChildren(this)` in a variable, so we can re-use it in the `navigateByArrowKey` call.